src/Pages/CreateMembers.jsx

```diff
@@ -1,30 +1,33 @@
+import { useState } from "react";
 import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
 
 export default function (props) {
   return (
     <section className="two-column-grid-forms">
       <div className="available-classes">
         <h2>Available Classes: </h2>
         <ul>
-          {props.map((oneClass, index) => {
-            const {} = props;
+          {/* {props.map((oneClass, index) => {
+            const {} = props; */}
```

Comment:
Might be a good idea to remove this commented code if it's not needed anymore :)

Reply:
this was part of the file setup
will delete as going forward 
thanks